src/wizard/apple/ios.md

```diff
@@ -123,3 +123,50 @@ SentrySDK.start { options in
     options.attachViewHierarchy = true
 }
 ```
+
+## Performance Monitoring for SwiftUI
+
+> This is an experimental feature
```

Comment:
`m`: Instead of duplicating the info, I would add a link to the instructions above. Same applies for getting started.